client/src/api/api.ts

```diff
@@ -120,6 +122,78 @@ export interface TopPeopleResult {
   url: string;
 }
 
+export interface SingleViewAnimeResult {
```

Comment:
I'd probably call this just Anime, given that this interface is used for the anime <Anime /> page, and describes all the data a singular anime returns.